debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java

```diff
@@ -119,21 +119,19 @@ public void execute(ChangeEventSourceContext context) throws InterruptedExceptio
                 int noMessageIterations = 0;
                 if (!stream.readPending(message -> {
```

Comment:
@gunnarmorling Lookign at the code I'd propose a small change in formatting.

Let's extract `stream.readPending` to set a local variable `messageProcessed` and then chane the `if` statement to logic

```
if (messageProcessed) {
// add comment
}
else {
}
```

I think it would improve reasoning about the code, WDYT?

Functionally I am fine with the change

Reply:
Yes, absolutely. I had been contemplating the same, but refrained from doing so in order to keep the diff small. Will push another commit for this.

Reply:
Done.